student.rb

```diff
@@ -1,12 +1,15 @@
-require_relative 'person'
-
-class Student < Person
-  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
-    super(age, parent_permission: parent_permission, name: name)
-    @classroom = classroom
-  end
-
-  def play_hooky
-    '¯(ツ)/¯'
-  end
-end
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
+    super(age, parent_permission: parent_permission, name: name)
+    @classroom = classroom
+    classroom&.add_student(self)
```

Comment:
- [ ] There should be a method on its own that handles association. Kindly remove this line 9 and add the association method.

  ```
  def classroom=(classroom)
    @classroom = classroom
    classroom.students.push(self) unless classroom.students.include?(self)
  end
  ```